bedrock/exp/templates/exp/firefox/lockwise.html

```diff
@@ -44,10 +44,10 @@
     ) %}
 
       <div class="mobile-download-buttons">
-        <a href="{{ lockwise_adjust_url('ios', 'lockwise-page') }}" data-link-type="download" data-download-os="iOS" class="ios-button" data-cta-text="apple-app-store" data-cta-type="lockwise-app-store" data-cta-position="primary">
+        <a href="{{ lockwise_adjust_url('ios', 'lockwise-page') }}" class="ios-button" data-cta-text="apple-app-store" data-cta-type="lockwise-app-store" data-cta-position="primary">
           <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="Download on the App Store" width="152" height="45">
         </a>
-        {{ google_play_button(href=lockwise_adjust_url('android', 'lockwise-page'), extra_data_attributes={'download-location': button_location, 'cta-type': 'lockwise-app-store', 'cta-text': 'google-play-store', 'cta-position': 'primary' } ) }}
+        {{ google_play_button(href=lockwise_adjust_url('android', 'lockwise-page'), product='lockwise', extra_data_attributes={'cta-type': 'lockwise-app-store', 'cta-text': 'google-play-store', 'cta-position': 'primary' } ) }}
```

Comment:
not a blocker- but maybe we should consider adding the option to add a param on adjust links to indicate if it is an experiment page.

Reply:
Perhaps, but that's not something we have any real specs for as these URLs are adjust links, and do not do FxA auth. It's not something related to fixing this bug, so I don't think it's worth complicating things here?